consensus/polybft/consensus_runtime.go

```diff
@@ -400,15 +410,13 @@ func (c *consensusRuntime) restartEpoch(header *types.Header) error {
 // for state sync events starting from given index and saves the message in database
 func (c *consensusRuntime) buildCommitment(epoch, fromIndex uint64) (*Commitment, error) {
 	// if it is not already built in the previous epoch
-	stateSyncEvents, err := c.state.getStateSyncEventsForCommitment(fromIndex, fromIndex+stateSyncMainBundleSize)
-	if err != nil {
-		if errors.Is(err, ErrNotEnoughStateSyncs) {
-			c.logger.Info("[buildCommitment] Not enough state syncs to build a commitment",
-				"epoch", epoch, "from state sync index", fromIndex)
-			// this is a valid case, there is not enough state syncs
-			return nil, nil
-		}
-
+	stateSyncEvents, err := c.state.getStateSyncEventsForCommitment(fromIndex, fromIndex+stateSyncMainBundleSize-1)
+	if len(stateSyncEvents) == 0 {
+		c.logger.Info("[buildCommitment] Not enough state syncs to build a commitment",
+			"epoch", epoch, "from state sync index", fromIndex)
+		// this is a valid case, there is not enough state syncs
```

Comment:
I'd just switch these two lines, for the sake of consistency.
```suggestion
                // this is a valid case, there is not enough state syncs
		c.logger.Info("[buildCommitment] Not enough state syncs to build a commitment",
			"epoch", epoch, "from state sync index", fromIndex)
```